nuxeo-retention-web/nuxeo-retention.html

```diff
@@ -117,6 +117,88 @@
   </template>
 </nuxeo-slot-content>
 
+<nuxeo-slot-content name="deleteDocumentAction" slot="DOCUMENT_ACTIONS" order="30" priority="1">
+  <template>
+    <nuxeo-filter
+      document="[[document]]"
+      user="[[user]]"
+      expression="this.hasPermission(document, 'MakeRecord')
+        && this.hasPermission(document, 'RemoveRecords')
+        && !nuxeo.retention.compliance.enabled"
```

Comment:
Normally, the mapping of a platform conf property with a UI conf property is done as follow:
The property values are stored in `Nuxeo.UI.config` so something like `Nuxeo.UI.config.retention.mode`?
Meaning we'll likely have something like
```
<property name="org.nuxeo.web.ui.retention.mode">${nuxeo.retention.mode}</property>
```
and we'll probably end up with
`&& Nuxeo.UI.config.retention.mode === 'governance'` in the filter?

Reply:
you are right, this was more of a placeholder. still, as we discussed, for now we're not implementing governance modes, so we'll probably rely on the existing boolean  **Nuxeo.UI.config.retention.compliance.enabled**